#this is the message received from the standard communication node for the act3d
float32[] inertia
float32[] damp
